docs/snowpark/snowpark-backend-joins.md

```diff
@@ -0,0 +1,178 @@
+# Join support in Snowpark
+
+Integrating data from disparate sources is a common necessity, often arising from the need to establish relationships between datasets originating from different sources. In the dynamic landscape of data analysis, the process of merging information plays a pivotal role in gaining comprehensive insights.
+
+If you wish to achieve this functionality in Morphir, you would implement something similar to the following example:
+
+```elm
+simpleInnerJoin : List TypeA -> List TypeB -> List {idA: Int, idB: Int}
+simpleInnerJoin dataSetA dataSetB =
+     dataSetA
+        |> innerJoin dataSetB
+            ( \a b -> a.id == b.id )
+        |> List.map (\ (x, y) -> 
+            { idA = x.id
+            , idB = y.id
+            })
+```
+
+The `simpleInnerJoin` example illustrates how to perform an *inner join* operation between two datasets, dataSetA and dataSetB, represented as lists of types TypeA and TypeB respectively. In this case, the join is based on a specific condition: equality between the `id` properties of elements in dataSetA and dataSetB.
+
+The `innerJoin` function (provided for the [Morphir.SDK](https://package.elm-lang.org/packages/finos/morphir-elm/latest/Morphir-SDK-List#leftJoin)) facilitates this join, taking the equality condition provided in the lambda function `(\a b -> a.id == b.id)`.
```

Comment:
Change the link to https://package.elm-lang.org/packages/finos/morphir-elm/latest/Morphir-SDK-List#innerJoin